aarch64_linux/aarch64_ci_build.sh

```diff
@@ -12,7 +12,7 @@ PATH=/opt/conda/bin:$PATH
 # Install OS dependent packages
 ###############################################################################
 yum -y install epel-release
-yum -y install less zstd libgomp
+yum -y install less zstd libgomp liblapack-dev
```

Comment:
Believe  liblapack-dev is for "Debian" based distros. Think you need lapack-devel from "RHEL" based distros as this is using pypi manylinux docker image to build.
ref: https://centos.pkgs.org/7/centos-x86_64/lapack-devel-3.4.2-8.el7.x86_64.rpm.html